frontend/src/features/Train/components/ImageUploadLayout.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+  DATA_SOURCE,
+  FileUploadData,
+  ImageUploadData,
+} from "@/features/Train/types/trainTypes";
+import { Button, Radio, Stack, Typography } from "@mui/material";
+import { Controller, UseFormReturn } from "react-hook-form";
+import dynamic from "next/dynamic";
+import { formatDate } from "@/common/utils/dateFormat";
+import {
+  useGetDatasetFilesDataQuery,
+  useUploadDatasetFileMutation,
+} from "@/features/Train/redux/trainspaceApi";
+import { DataGrid } from "@mui/x-data-grid";
+import prettyBytes from "pretty-bytes";
+import React from "react";
+
+const FilerobotImageEditor = dynamic(
+  () => import("react-filerobot-image-editor"),
+  {
+    ssr: false,
+  }
+);
+
+const TABS = dynamic(
```

Comment:
TABS and TOOLS don't have to be dynamically imported (seems to give a weird typing error when it is)

Reply:
I still get the import error when I import it normally (it happens when I refresh). Looking for a way to resolve the weird type error, but it doesn't seem to cause any issues when I run it.